__tests__/submission.test.ts

```diff
@@ -0,0 +1,421 @@
+import { expect } from '@jest/globals'
+import { buildSchema } from 'graphql'
+import { IMockServer, mockServer, addMocksToSchema } from '@graphql-tools/mock'
+import { addResolversToSchema } from '@graphql-tools/schema'
+import { initiatilizeFirebaseInstance } from '../src/firebaseDb'
+import resolvers from '../src/resolvers'
+import { Submission } from '../src/typeDefs/dbSchema'
+import { ErrorCode } from '../src/result'
+import fs from 'fs'
+import path from 'path'
+// THIS IMPORT IS NEEDED WHEN YOU WANT TO CLEAR THE DATA FROM EMULATOR
+// import { initializeTestEnvironment, RulesTestEnvironment } from '@firebase/rules-unit-testing'
+// Please install this with @firebase/rules-unit-testing
+
+const schema = buildSchema(fs.readFileSync(
+    path.join(__dirname, '../src/typeDefs/schema.graphql'),
+    'utf-8'
+))
+
+const submissionByIdQueryData = {
+    query: `query GetSubmissionByID($submissionId: ID!) {
+        submission(id: $submissionId) {
+            id
+            googleMapsUrl
+            healthcareProfessionalName
+            spokenLanguages {
+              iso639_3
+              nameJa
+              nameEn
+              nameNative
+            }
+            isUnderReview
+            isApproved
+            isRejected
+        }
+    }`,
+    variables: {
+        submissionId: '1'
+    }
+}
+
+describe('query getSubmissionById', () => {
+    let server: IMockServer
+
+    // TO DO: CLEAR (EMULATOR) DATABASE AFTER TESTS
```

Comment:
This is something we should add in a next PR, at the moment I get an error but could not figure out why. I need to look into it a bit more. 

Error:
**The host and port of the database emulator must be specified. (You may wrap the test script with `firebase emulators:exec './your-test-script'` to enable automatic discovery, or specify manually via initializeTestEnvironment({database: {host, port}}).**